automation-tests/tests/homePage.test.ts

```diff
@@ -0,0 +1,17 @@
+//import { test, expect, selectors } from '@playwright/test';
+import { test, expect } from "../tests/utils/fixtures";
+
+test('Carrot Home page', async ({ page }) => {
+    await test.step('Navigate to Home page', async () => {
+        await page.goto('https://app.staging.carrot.community/#/?chain=sepolia');
```

Comment:
The `goto` should  use the configured `baseUrl`, no?

Reply:
yes, done